main.cpp

```diff
@@ -284,11 +284,11 @@ int calculatePixelCount(int numSrcBytes, color_format_t const* colorFormat, inde
 
 void applyTileLayout(rgba8888_t* buffer, int width, int height, int tileWidth, int tileHeight) {
 	if ((width % tileWidth) != 0) {
-		printf("Chosen layout must have a width divisible by %d.\n", tileWidth);
+    eprintf("Chosen layout must have a width divisible by %d.\n", tileWidth);
```

Comment:
can you fix the indentation here please? (prefer tabs over spaces)

Reply:
done